tests/test_query.py

```diff
@@ -173,6 +173,59 @@ def test_get_active_user_requests(client):
     assert res.json() == [user_request]
 
 
+def test_get_filtered_user_requests(client):
+    fake_jwt = "1.2.3"
+    filtered_requests = []
+    # create a request with policy_id foo, status = APPROVED and revoke = False
+    data = {
+        "policy_id": "foo",
+        "resource_id": "draft_uniqid",
+        "revoke": "False",
+        "resource_display_name": "My Draft Resource",
+        "status": "APPROVED",
+    }
+    res = client.post(
+        "/request", json=data, headers={"Authorization": f"bearer {fake_jwt}"}
+    )
+    assert res.status_code == 201, res.text
+    filtered_requests.append(res.json())
+
+    # create a request with policy_id bar, status = APPROVED and revoke = False
+    data = {
+        "policy_id": "bar",
+        "resource_id": "active_uniqid",
+        "revoke": "False",
+        "resource_display_name": "My Active Resource",
+        "status": "APPROVED",
+    }
+    res = client.post(
+        "/request", json=data, headers={"Authorization": f"bearer {fake_jwt}"}
+    )
+    assert res.status_code == 201, res.text
+    filtered_requests.append(res.json())
+
+    # create a request with a different policy_id and status but with revoke=False
+    data = {
+        "policy_id": "random",
+        "resource_id": "final",
+        "resource_display_name": "My Final Resource",
+        "revoke": "False",
+        "status": "CANCELLED",
+    }
+    res = client.post(
+        "/request", json=data, headers={"Authorization": f"bearer {fake_jwt}"}
+    )
+    assert res.status_code == 201, res.text
+
+    # check that only get those requests which match the filtered criteria
+    res = client.get(
+        "/request/user?policy_id=foo&policy_id=bar&revoke=False&status=APPROVED",
```

Comment:
i think this implementation works for all the fields we have [here](https://github.com/uc-cdis/requestor/blob/6a3095b/src/requestor/models.py#L24-L34), except maybe `created_time` and `updated_time` - could you add a test case, check if it works, and if it doesn't, handle it by either fixing the implementation or returning a 400?

Reply:
Should it work or not work with `created_time` and `updated_time`? I've added a test case with `created_time` and it works. How should I proceed from here?

Reply:
if it works, good :)

Reply:
i just don't want it to crash and return 500. 400 or 200 are both good solutions